src/docker-compose.yml

```diff
@@ -1,16 +1,19 @@
 version: '3.7'
 
 services:
+
   db:
     container_name: paws-compose-db
     ports:
       - "5432:5432"
-    image: postgres:13.2-alpine
+    image: postgres:15.4-alpine
     volumes:
-      - postgres:/var/lib/postgresql/data
+      - postgres15:/var/lib/postgresql15/data
```

Comment:
Just for my own learning, can you tell me why we need to update volume name and directory here?

Reply:
That was an attempt to preserve the old pg13 install/data to make moving easier